web/03-inputs.php

```diff
@@ -308,6 +312,33 @@ function mapsLoaded() {
     // create the tooltip and its text
     var info="<b><?php echo $siteinfo['sitename']; ?></b><br />";
     info+="<?php echo $siteinfo['city']; ?>, <?php echo $siteinfo['state']; ?>, <?php echo $siteinfo['country']; ?><br/>";
+    if ("<?php echo $siteinfo['mat']; ?>") {
+    	info+="Mean Annual Temp: <?php echo $siteinfo['mat']; ?><br/>";
+    }
+    if ("<?php echo $siteinfo['map']; ?>") {
```

Comment:
probably easier just to have the if inside the php
```
<?php
if (isset($siteinfo, 'map') {
  echo 'info+="Mean Annual Precip: ' . $siteinfo['map'] . '<br/>";';
}
?>
```
This way the javascript will only execute the code that is actually needed.